pkg/sql/opaque.go

```diff
@@ -55,13 +55,11 @@ func buildOpaque(
 		}
 		// TODO (Chengxiong): Remove this version gate in 22.2
 		if evalCtx.Settings.Version.IsActive(ctx, clusterversion.EnableDeclarativeSchemaChanger) {
```

Comment:
While you're at it could you remove this version gate please?

Reply:
done!

Reply:
edit: I reverted this change back because we shouldn't remove this version gate yet.